mmdet/core/hook/yolox_mode_switch_hook.py

```diff
@@ -0,0 +1,33 @@
+from mmcv.parallel import is_module_wrapper
+from mmcv.runner.hooks import HOOKS, Hook
+
+
+@HOOKS.register_module()
+class YOLOXModeSwitchHook(Hook):
+    """Switch the mode of YOLOX during training.
+
+    This hook turns off the mosaic and mixup data augmentation and switches
+    to use L1 loss in bbox_head.
+
+    Args:
+        last_epoch (int): The last epoch of close data augmentation and
```

Comment:
Switch the mode at `last_epoch` epochs from the end of the training

Reply:
I think `num_epochs` or `num_last_epochs` could be better, the docstring can be like the number of later epochs in the end of the training to close the data augmentation and switch to L1 loss.

Reply:
@ZwwWayne , Done as suggested here.